packages/paste-core/components/tabs/src/Tab.tsx

```diff
@@ -85,13 +113,13 @@ const getTabBoxStyles = (orientation: Orientation, variant: Variants): BoxStyleP
         },
         /* Hover states */
         _hover: {
-          color: isInverse ? "colorTextInverse" : "colorTextPrimary",
-          borderBottomColor: isInverse ? "colorBorderInverseStrongest" : "colorBorderPrimary",
+          color: isInverse ? "colorTextInverse" : "colorTextPrimaryStrongest",
         },
         _selected_hover: {
           borderColor: isInverse ? "colorBorderInverseStrongest" : "colorBorderWeak",
           color: isInverse ? "colorTextInverse" : "colorTextPrimaryStronger",
           borderBottomColor: isInverse ? "colorBorderInverseWeakest" : "colorBorderWeakest",
+          boxShadow: isInverse ? "shadowBorderTopPrimary" : "shadowBorderTopPrimaryStronger",
```

Comment:
- rename shadowBorderTopPrimaryStronger to shadowBorderTopPrimaryStrongest
- rename shadowBorderInverseTopPrimary to shadowBorderTopInverseStrongest